tools/istio-docker.mk

```diff
@@ -262,6 +262,18 @@ docker.push: $(DOCKER_PUSH_TARGETS)
 docker.basedebug:
```

Comment:
TODO: someone familiar with the release scripts should check if 'make docker.basedebug' is run as part of the release builds.